TestModels/Union/runtimes/rust/src/operation.rs

```diff
@@ -0,0 +1,7 @@
+// Code generated by software.amazon.smithy.rust.codegen.smithy-rs. DO NOT EDIT.
+
+/// Types for the `GetUnion` operation.
+pub mod get_union;
+
+/// Types for the `GetUnionKnownValue` operation.
+pub mod get_union_known_value;
```

Comment:
Technically wrong since the Smithy operation is `GetKnownValueUnion`, but I really don't want to make you fix it manually. :) I think it will become obvious and trivial to fix when we move to actual code generation so I don't want to block on such typos in general.